diff --git a/modules/skcms/BUILD.gn b/modules/skcms/BUILD.gn
index 5d037a94c9..929c3105fc 100644
--- a/modules/skcms/BUILD.gn
+++ b/modules/skcms/BUILD.gn
@@ -47,7 +47,7 @@ arch("skcms_TransformHsw") {
     cflags = [
       "-mavx2",
       "-mf16c",
-      "-std=c11",
+      "-std=c++17",
     ]
   }
 }
@@ -75,7 +75,7 @@ arch("skcms_TransformSkx") {
       "-mavx512cd",
       "-mavx512bw",
       "-mavx512vl",
-      "-std=c11",
+      "-std=c++17",
     ]
   }
 }
@@ -83,7 +83,7 @@ arch("skcms_TransformSkx") {
 static_library("skcms") {
   cflags = []
   if (!is_win || is_clang) {
-    cflags += [ "-std=c11" ]
+    cflags += [ "-std=c++17" ]
   }
   if (target_cpu != "x64" || target_os == "android") {
     defines = [
